partiql-ast/src/main/java/org/partiql/ast/expr/ExprPath.java

```diff
@@ -19,22 +18,20 @@ public final class ExprPath extends Expr {
     @NotNull
     private final Expr root;
 
-    @Nullable
-    private final PathStep next;
+    @NotNull
+    private final List<PathStep> steps;
```

Comment:
(self-review) `ExprPath` modeling changed to be more alike the EXCLUDE path modeling. The prior linked list approach could be slightly annoying to use compared to just a normal Java list.